xbmc/video/VideoDatabase.cpp

```diff
@@ -4566,6 +4589,13 @@ void CVideoDatabase::UpdateTables(int iVersion)
       }
     }
   }
+  if (iVersion < 94)
```

Comment:
Yeah, I meant devs with alpha users :)
Forgot about the indices and statistics for a moment. And I was aware of every iteration of the update logic being important, not sure what I was thinking.
Thank you for explaining!